packages/mobile/.env

```diff
@@ -1,11 +1,11 @@
 ENVIRONMENT=local
-DEFAULT_TESTNET=pilot
+DEFAULT_TESTNET=alfajoresstaging
 # If ZERO_SYNC_ENABLED_INITIALLY, local geth will not run initially. 
 # If toggled on, it will use DEFAULT_SYNC_MODE. See src/geth/consts.ts for more info
-ZERO_SYNC_ENABLED_INITIALLY=true
+ZERO_SYNC_ENABLED_INITIALLY=false
 DEFAULT_SYNC_MODE=5
 DEV_SETTINGS_ACTIVE_INITIALLY=true
 FIREBASE_ENABLED=true
 SECRETS_KEY=debug
-SHOW_TESTNET_BANNER=true
+SHOW_TESTNET_BANNER=false
```

Comment:
Sure you want to merge these too?

Reply:
The zero sync was intentional but the banner was not. will revert.